package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hs-web-team/eslint-config-node",
-  "version": "0.3.1",
+  "version": "1.0.0",
```

Comment:
Maybe we should just do a minor version, we are still in version `0` so breaking changes are ok here

Reply:
Makes sense 🙌, I've just changed that to be a minor version in the last commit